os/booting-on-cloudca.md

```diff
@@ -0,0 +1,260 @@
+# Running CoreOS on cloud.ca
+
+[cloud.ca](http://www.cloud.ca) is a regional self-service IaaS platform
```

Comment:
Could we reduce the marketing feel of this graf?

Maybe:
[cloud.ca]() is a regional IaaS designed to help meet regulatory or security requirements by storing data in Canada. CoreOS is a supported operating system for VMs on the cloud.ca infrastructure.